### ISSUE TITLE
TestNG support?

### ISSUE THREAD

**Issue by washtubs** [mfussenegger/nvim-jdtls#30]
I was fiddling with the debug stuff and I think I set everything up correctly, but when I ran it in a testNG class, I saw that it was throwing an error in:

    M.execute_command(cmd_junit_args, function(err1, launch_args)
      if err1 then
        print('Error retrieving launch arguments: ' .. (err1.message or vim.inspect(err1)))
        return
      end
      local args = table.concat(launch_args.programArguments, ' ');

Basically programArguments was nil. Also looking at the var saying junit I suspect that's where the problem lies. If there's currently no support, maybe we should get a message for the user explaining

**Comment by mfussenegger** [comment]
Yes you're right, TestNG is currently not supported.

We could check the kind and raise a better error if it is `TestKind.TestNG`:

https://github.com/mfussenegger/nvim-jdtls/blob/357d0b405235e3dfb0b15450f33ad6d10cbf2122/lua/jdtls.lua#L872-L883

Or even better, add support for it.

Some of the logic is inspired by how vscode-java-test does things, and they also contain components for TestNG: https://github.com/microsoft/vscode-java-test/tree/master/src/runners/testngRunner 

Not sure how difficult it would be to extend the support in nvim-jdtls.

**Comment by mfussenegger** [comment]
testng tests should work now with https://github.com/mfussenegger/nvim-jdtls/pull/58
